### PR TITLE
feat(ui): Add arrows to each level of dropdown [WIP]

### PR DESCRIPTION
**Pull request by mikellykels**
Add arrows to each level of the Issue Action Button dropdowns on the Issue Details page and Issue Stream. Also started removing bootstrap styles and styling the components.

[WOR-455](https://getsentry.atlassian.net/browse/WOR-455)

### Before
Screen Shot 2020-12-04 at 1 53 52 PM
Screen Shot 2020-12-04 at 1 54 14 PM
Screen Shot 2020-12-04 at 2 38 43 PM

### After
Screen Shot 2020-12-04 at 1 47 40 PM
Screen Shot 2020-12-04 at 1 48 05 PM
Screen Shot 2020-12-04 at 1 48 12 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/actions/ignore.tsx [hunk 1]
```diff
@@ -276,48 +313,192 @@ export default class IgnoreActions extends React.Component<Props, State> {
...
 
 const StyledDropdownLink = styled(DropdownLink)`
   transition: none;
+  border-top-left-radius: 0 !important;
+  border-bottom-left-radius: 0 !important;
+`;
+
+const DropdownMenuItem = styled('li')`
+  :not(:first-child) {
+    border-top: 1px solid ${p => p.theme.border};
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/actions/ignore.tsx:449]: Should this be innerBorder? The screenshots seem high contrast in the menus.

Comment (mikellykels) [reply src/sentry/static/sentry/app/components/actions/ignore.tsx:449]: This is what it looks like with `innerBorder` (the designs had this set for `border`) - @robinrendle thoughts?

Screen Shot 2020-12-10 at 1 31 57 AM
Screen Shot 2020-12-10 at 1 32 12 AM

Comment (markstory) [reply src/sentry/static/sentry/app/components/actions/ignore.tsx:449]: I prefer the inner border as it is more subtle and looks like other striped dropdowns in the app (global selection header, saved issue search etc).

Comment (mikellykels) [reply src/sentry/static/sentry/app/components/actions/ignore.tsx]: cool, updated to innerBorder

File: src/sentry/static/sentry/app/components/actions/ignore.tsx [hunk 2]
```diff
@@ -276,48 +313,192 @@ export default class IgnoreActions extends React.Component<Props, State> {
...
+      }
+    }
+  }
+  &:hover > span {
+    background: ${p => p.theme.bodyBackground};
+    border-radius: ${p => p.theme.borderRadius};
+  }
+`;
+
+const StyledMenuItem = styled(MenuItem)`
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/actions/ignore.tsx:472]: You have the same styled components in a both the ignored and resolve modules. Could they be consolidated (as a follow up pull request if you'd prefer)?

File: src/sentry/static/sentry/app/components/actions/ignore.tsx [hunk 3]
```diff
@@ -276,48 +313,192 @@ export default class IgnoreActions extends React.Component<Props, State> {
...
+    border-top: 1px solid ${p => p.theme.innerBorder};
+  }
+  > span {
+    border-radius: ${p => p.theme.borderRadius};
+    display: block;
+    > ul {
+      border-radius: ${p => p.theme.borderRadius};
+      top: 0;
+      left: 100%;
+      margin-top: -5px;
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/actions/ignore.tsx:458]: Should this be 0? With 5px the submenus are staggered from the item that spawned them.

**Review by markstory — APPROVED** [review]

File: src/sentry/static/sentry/app/components/actions/ignore.tsx (hunk 1 above)
Comment (robinrendle) [reply src/sentry/static/sentry/app/components/actions/ignore.tsx]: Yeah, this looks way better. ✨

File: src/sentry/static/sentry/app/components/actions/ignore.tsx (hunk 3 above)
Comment (mikellykels) [reply src/sentry/static/sentry/app/components/actions/ignore.tsx:458]: Adjusted 👍 I think I copied over the original style which had this set at 0 and didn't realize the new designs had them staggered this way instead

File: src/sentry/static/sentry/app/components/actions/ignore.tsx (hunk 2 above)
Comment (mikellykels) [reply src/sentry/static/sentry/app/components/actions/ignore.tsx:472]: Ah yeah, I wanted to try and consolidate the styles in a separate PR 👍

**Pull request merged**